### PR TITLE
Spelling; line-lengths, other cosmetic changes

### PR DESCRIPTION
**Pull request by lukesonnet**

### CODE REVIEW

File: R/estimatr_lm_robust.R [hunk 1]
```diff
@@ -238,7 +238,8 @@ lm_robust <- function(formula,
     yoriginal <- model_data[["outcome"]]
     Xoriginal <- model_data[["design_matrix"]]
     model_data <- demean_fes(model_data)
-    attr(model_data$fixed_effects, "fe_rank") <- sum(model_data[["fe_levels"]]) + 1
+    attr(model_data$fixed_effects, "fe_rank") <-
```
Comment (nfultz) [inline R/estimatr_lm_robust.R]: this should be one line

File: R/helper_clean_model_data.R [hunk 2]
```diff
@@ -165,7 +167,11 @@ demean_fes <- function(model_data) {
     X = model_data[["design_matrix"]],
     Zmat = model_data[["instrument_matrix"]],
     fes = model_data[["fixed_effects"]],
-    weights = if (is.numeric(model_data[["weights"]])) model_data[["weights"]] else rep(1, nrow(model_data[["design_matrix"]])),
+    weights =
```
Comment (nfultz) [inline R/helper_clean_model_data.R]: `weights = if(...)\n`

File: R/helper_lm_robust_fit.R [hunk 3]
```diff
@@ -192,10 +197,20 @@ lm_robust_fit <- function(y,
...
+          if (se_type %in% c("HC2", "HC3", "CR2") && fes)
+            cbind(data[["X"]], data[["femat"]])
+          else data[["X"]],
+        Xunweighted =
+          if (se_type %in% c("HC2", "HC3", "CR2") && fes && weighted)
+            cbind(data[["Xunweighted"]], data[["fematunweighted"]])
+          else data[["Xunweighted"]],
         XtX_inv = fit$XtX_inv,
-        ei = if (se_type == "CR2" && weighted) fit_vals[["ei.unweighted"]] else fit_vals[["ei"]],
+        ei =
```
Comment (nfultz) [inline R/helper_lm_robust_fit.R]: `ei = if()\n`

File: R/S3_predict.R [hunk 4]
```diff
@@ -125,8 +125,11 @@ predict.lm_robust <- function(object,
 
     ret <- list()
 
-    var_fit <-
-      apply(X[, !beta_na, drop = FALSE], 1, function(x) tcrossprod(crossprod(x, object$vcov), x))
+    var_fit <- apply(
+      X[, !beta_na, drop = FALSE],
+      1,
+      function(x) tcrossprod(crossprod(x, object$vcov), x)
```
Comment (nfultz) [inline R/S3_predict.R]: double checking ,do we really want `(t(x) V) t(x)` or are you relying on implicit vector-matrix conversions.

`outer(x,x) * object$vcov` is a little more obvious if that's what you really want

Comment (lukesonnet) [reply R/S3_predict.R]: I actually want x'Vx where x is a column vector, so I'm relying on the implicit conversion.

**Pull request merged**